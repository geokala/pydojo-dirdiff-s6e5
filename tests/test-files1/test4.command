./dirdiff tests/test-files1/test4.{a,b}
