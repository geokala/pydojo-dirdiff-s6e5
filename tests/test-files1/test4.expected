2 difference between directories:
test4.a contains c.txt that cannot be found in test4.b
test4.b contains a.txt that cannot be found in test4.b
